dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/DependencyList.java

```diff
@@ -16,146 +16,89 @@
  */
 package org.apache.camel.dsl.jbang.core.commands;
 
-import java.io.File;
+import java.nio.file.Path;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
-import java.util.Properties;
+import java.util.Set;
+import java.util.TreeSet;
 
-import javax.xml.parsers.DocumentBuilder;
-import javax.xml.parsers.DocumentBuilderFactory;
-
-import org.w3c.dom.Document;
-import org.w3c.dom.Element;
-import org.w3c.dom.NodeList;
-
-import org.apache.camel.dsl.jbang.core.common.CommandLineHelper;
-import org.apache.camel.dsl.jbang.core.common.RuntimeUtil;
-import org.apache.camel.dsl.jbang.core.common.XmlHelper;
+import org.apache.camel.dsl.jbang.core.common.MavenGavComparator;
+import org.apache.camel.main.KameletMain;
+import org.apache.camel.main.download.DownloadListener;
 import org.apache.camel.tooling.maven.MavenGav;
-import org.apache.camel.util.CamelCaseOrderedProperties;
-import org.apache.camel.util.FileUtil;
 import picocli.CommandLine;
 
-@CommandLine.Command(name = "list",
-                     description = "Displays all Camel dependencies required to run")
-public class DependencyList extends Export {
+@CommandLine.Command(name = "list", description = "Displays all Camel dependencies required to run")
+public class DependencyList extends CamelCommand {
+
+    // ignored list of dependencies, can be either groupId or artifactId
+    private static final String[] SKIP_DEPS = new String[] {
+            "org.fusesource.jansi", "org.apache.logging.log4j", "camel-core-languages", "camel-endpointdsl",
+            "camel-java-joor-dsl" };
```

Comment:
Why are there ignored dependencies?

Reply:
The org.fusesource.jansi, org.apache.logging.log4j were already there before and I didn't want to disrupt it. The other ignored dependencies are mostly from camel core modules, I think these could be removed from the list.

Reply:
I removed the `camel-*` artifacts from the ignored list.